definitions/install.yaml

```diff
@@ -736,9 +736,16 @@ spec:
           args:
           - "--filterK8Resources=[Event,*,*][*,kube-system,*][*,kube-public,*][*,kube-node-lease,*][Node,*,*][APIService,*,*][TokenReview,*,*][SubjectAccessReview,*,*][*,kyverno,*][Binding,*,*][ReplicaSet,*,*]"
           # customize webhook timout
-          # - "--webhooktimeout=4"
+          #- "--webhooktimeout=4"
           ports:
           - containerPort: 443
           env:
           - name: INIT_CONFIG
             value: init-config
+          resources:
+            requests:
+              memory: "64Mi"
+              cpu: "250m"
+            limits:
+              memory: "128Mi"
+              cpu: "500m"
```

Comment:
we should not configure a CPU limit. See:

https://www.reddit.com/r/kubernetes/comments/all1vg/on_kubernetes_cpu_limits/

https://github.com/kubernetes/kubernetes/issues/51135

Reply:
CPU is a preemptible resource. So setting a request is a good idea, but setting a limit rarely has the intended effect.